scripts/prepare-publish.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs");
```

Comment:
Can we add a bit of context around why we need this script as a comment in this module?

Reply:
Good idea!